qa/workunits/rbd/cli_generic.sh

```diff
@@ -1264,6 +1264,12 @@ test_mirror_snapshot_schedule() {
     rbd mirror snapshot schedule add 1h 00:15
     test "$(rbd mirror snapshot schedule ls)" = 'every 1h starting at 00:15:00'
 
+    # Negative tests
+    expect_fail rbd mirror snapshot schedule add dummy
+    expect_fail rbd mirror snapshot schedule add -p rbd2/ns1 --image test1 dummy
+    expect_fail rbd mirror snapshot schedule remove dummy
+    expect_fail rbd mirror snapshot schedule remove -p rbd2/ns1 --image test1 dummy
+    test "$(rbd mirror snapshot schedule ls -R --format json)" != "[]"
```

Comment:
Replace this `test` with two, explicitly stating what is expected at each level and add a line break before `rbd rm`:
```suggestion
    test "$(rbd mirror snapshot schedule ls)" = 'every 1h starting at 00:15:00'`
    test "$(rbd mirror snapshot schedule ls --image rbd2/ns1/test1)" = 'every 1m'
    
```

Reply:
IIUC `rbd trash purge schedule` is also affected by this bug.  Please add a similar test there too.

Reply:
The commit and PR titles also need to be updated to not state that it's specific to `rbd mirror snapshot schedule`.

Reply:
Done, thanks

Reply:
Not quite -- still need to update commit and PR titles.